manifests/hash.pp

```diff
@@ -50,6 +50,4 @@
     path    => $name,
     mode    => $mode,
   }
-
-  Class['postfix'] -> Postfix::Hash[$title]
```

Comment:
This seems unrelated and should probably be its own PR.

Reply:
I needed this to get postfix::hash to loop properly.